workflows/proteomics/clinicalmp/clinicalmp-verification/iwc_clinicalmp_verification_workflow_tests.yml

```diff
@@ -0,0 +1,49 @@
+- doc: Test outline for iwc_clinicalmp_verification_workflow
+  job:
+    SGPS_peptide-report:
```

Comment:
These correspond to workflow inputs and outputs and are shown to users in the workflow invocation view and report. Can you replace `-` and `_` with whitespaces ? Please also replace these in the workflow itself, the labels need to match.

Reply:
Replaced _ and - with whitespaces in yml file and also labels and newname in .ga file.